escrow/control_service.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package escrow;
+
+service ProviderControlService {
+
+    //get list of all unclaimed "payments".
+    //in PaymentsSignatureReply signatures MUST be omited
+    //if signature field is present then response should be considered as invalid
+    rpc GetListUnclaimed(GetPaymentsListRequest)  returns (PaymentsSignatureReply) {}
```

Comment:
Did you intend PaymentLisReply ? , I dont see the definition of PaymentSignatireReply